ar_exercise/exercise.rb

```diff
@@ -34,38 +34,108 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.first or (Tenant.find(1))
+
 # get all tenants older than 65
+older_tenants = Tenant.where("age > 65")
+
 # get all apartments whose price is greater than $2300
+expensive_apartments = Apartment.where("monthly_rent > 2300", :rent)
```

Comment:
The second argument here is no necessary since you're not subbing in a value into a string.

```ruby
# this works
seniors = Tenant.where("age > 65")

# if you need a variable for the age it would look like this:
age_to_check = 50
seniors = Tenant.where("age > ?", age_to_check)
```